lib/components/Stepper/Stepper.test.tsx

```diff
@@ -190,6 +190,8 @@ describe('Stepper', () => {
       await userEvent.keyboard('{arrowleft}');
       expect(secondStep).toHaveFocus();
       await userEvent.tab();
+      expect(thirdStep).toHaveFocus();
```

Comment:
Did we change something to give the Stepper an extra step?

Reply:
I havent been able to put my finger on this one, why this is passing in master has me confused as the test now replicates what happens in the browser (both production and this branch).

Will have a poke around and see if i can provide some more clarity.